vismne/io.py

```diff
@@ -31,3 +37,63 @@ def _get_subjects_dir(subjects_dir=None, raise_error=True):
                          % subjects_dir)
 
     return subjects_dir
+
+
+def read_scalar_data(filepath):
+    """Load in scalar data from an image.
+
+    Parameters
+    ----------
+    filepath : str
+        path to scalar data file
+
+    Returns
+    -------
+    scalar_data : numpy array
+        flat numpy array of scalar data
+    """
+    try:
```

Comment:
usually I try to avoid try-catch constructs when I can ... errors can pass silently with them